### PR TITLE
GAWB-2357 changing keyword

### PR DESCRIPTION
**Pull request by UrsaStutsman**

### CODE REVIEW

File: src/cljs/main/broadfcui/common/entity_table.cljs [hunk 1]
```diff
@@ -116,7 +116,7 @@
                :on-row-click (:on-row-click props)
                :on-column-change (:on-column-change props)}
               :toolbar
-              {:items
+              {:get-items
                (fn [table-props]
                 (cons [comps/FilterGroupBar
                        {:filter-groups (map (fn [entity-type]
```
Comment (s-rubenstein) [inline src/cljs/main/broadfcui/common/entity_table.cljs:122]: Below, there is a use of :toolbar-items. (Line 134.)

File: src/cljs/main/broadfcui/page/library/library_page.cljs [hunk 2]
```diff
@@ -76,7 +76,7 @@
                   :header-cell {:padding "0.5rem 0 0.5rem 1rem"}
                   :body-cell {:padding "0.3rem 0 0.3rem 1rem"}}}
          :toolbar ;; FIXME: magic numbers below:
-         {:items (constantly
+         {:get-items (constantly
```
Comment (blrubenstein) [inline src/cljs/main/broadfcui/page/library/library_page.cljs]: This file also makes use of a ':selected-items'. Is that something that should be changed to ':get-selected-items'?

File: src/cljs/main/broadfcui/common/entity_table.cljs (hunk 1 above)
Comment (UrsaStutsman) [reply src/cljs/main/broadfcui/common/entity_table.cljs:122]: Fixed that and similar problem on table.

**Review by s-rubenstein — APPROVED** [review]

File: src/cljs/main/broadfcui/common/entity_table.cljs [hunk 3]
```diff
@@ -131,7 +131,7 @@
                                               :selected-entity-type selected-entity-type)
                                        (when-let [f (:on-entity-type-selected props)]
                                          (f selected-entity-type))))}]
-                      (when-let [toolbar-items (:toolbar-items props)]
+                      (when-let [toolbar-items (:get-toolbar-items props)]
```
Comment (zarsky-broad) [inline src/cljs/main/broadfcui/common/entity_table.cljs]: Can you just match the let name to the keyword here? (`get-toolbar-items`)

File: src/cljs/main/broadfcui/common/table/table.cljs [hunk 4]
```diff
@@ -104,7 +104,7 @@
                                  {:initial-text (:filter-text query-params)
                                   :on-filter #(swap! state update :query-params assoc :filter-text % :page-number 1)}
                                  (:inner filter-bar-props))]]))
-         (when-let [items (:items toolbar)]
+         (when-let [items (:get-items toolbar)]
```
Comment (zarsky-broad) [inline src/cljs/main/broadfcui/common/table/table.cljs]: As above, match name to keyword

**Review by zarsky-broad — APPROVED** [review]

**Pull request merged**